examples/generative/deep_dream.py

```diff
@@ -38,8 +38,8 @@
 
 import numpy as np
 import tensorflow as tf
-from tensorflow import keras
-from tensorflow.keras.applications import inception_v3
+import keras
```

Comment:
This example is TF only, we should set the TF backend via `os.environ`

Reply:
Done.